molecule/firewall_rules/converge.yml

```diff
@@ -3,38 +3,1154 @@
   hosts: all
   become: true
   tasks:
+      # Test basic functionality with different actions
+    - name: "Action: Test pass action"
+      puzzle.opnsense.firewall_rules:
+        interface: 'lan'
+        action: 'pass'
+        description: "New Test pass Rule"
+
+    - name: "Action: Test block action"
+      puzzle.opnsense.firewall_rules:
+        interface: 'lan'
+        action: 'block'
+        description: "New Test block Rule"
+
+    - name: "Action: Test reject action"
+      puzzle.opnsense.firewall_rules:
+        interface: 'lan'
+        action: 'reject'
+        description: "New Test reject Rule"
+
+      # Test basic functionality of the disabled button
+    - name: "Disabled: Test disabled button"
+      puzzle.opnsense.firewall_rules:
+        interface: 'lan'
+        action: 'pass'
+        description: "New Test disabled pass Rule"
+
```

Comment:
disabled missing!

Reply:
added in https://github.com/DonGiovanni83/puzzle.opnsense/pull/3/commits/550021dda41eae7366ae0f6c764a9fcc2fa91f7f.